.github/workflows/e2e.yml

```diff
@@ -135,3 +135,63 @@ jobs:
           IMAGE_TAG=e2e KLUSTERLET_DEPLOY_MODE=Singleton make test-e2e
         env:
           KUBECONFIG: /home/runner/.kube/config
+  e2e-multiplehubs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: add permisson to docker.sock
+        run: sudo chown runner:docker /var/run/docker.sock
+        if: ${{ env.ACT }} # this step only runs locally when using the
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - name: install imagebuilder
+        run: go install github.com/openshift/imagebuilder/cmd/imagebuilder@v1.2.3
+      - name: Build images
+        run: IMAGE_TAG=e2e make images
+      - name: Setup kind cluster
```

Comment:
setup environment in action would be a bit hard for local dev testing. We should probably prepare a script in hack to setup the local multi-hubs environment.

Reply:
Agree, the script for local-test is added in hack dir.